cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh

```diff
@@ -20,4 +20,4 @@
 # For systems without journald
 mkdir -p /var/log/journal
 
-exec /usr/local/bin/fluentd $FLUENTD_ARGS
+exec /usr/local/bin/fluentd "$FLUENTD_ARGS"
```

Comment:
```suggestion
exec /usr/local/bin/fluentd "${FLUENTD_ARGS}"
```

Reply:
Actually, please don't make this change at all, please see:  https://github.com/kubernetes/kubernetes/pull/74187#issuecomment-465870299

We need to split these args.